docs/business.yaml

```diff
@@ -503,6 +503,196 @@ paths:
+                              addressCity: Richmond
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: V7E 1V2
+                              streetAddress: 10300 4th Ave
+                              streetAddressAdditional: ''
+                            officer:
+                              firstName: Doe
+                              lastName: Smith
+                              middleInitial: ''
+                              prevFirstName: ''
+                              prevLastName: ''
+                              prevMiddleInitial: ''
+                          - actions:
+                            - addressChanged
+                            appointmentDate: '2024-07-09'
+                            cessationDate: null
+                            deliveryAddress:
+                              addressCity: Burnaby
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: V5C 3V8
+                              streetAddress: 3765 1st Ave
+                              streetAddressAdditional: ''
+                            mailingAddress:
+                              addressCity: Burnaby
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: V5C 3V8
+                              streetAddress: 3765 1st Ave
+                              streetAddressAdditional: ''
+                            officer:
+                              firstName: Jane
+                              lastName: Smith
+                              middleInitial: ''
+                              prevFirstName: ''
+                              prevLastName: ''
+                              prevMiddleInitial: ''
+                          - actions:
+                            - nameChanged
+                            appointmentDate: '2024-07-09'
+                            cessationDate: null
+                            deliveryAddress:
+                              addressCity: Victoria
+                              addressCountry: CA
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V8W 1W4
+                              streetAddress: 1-940 Blanshard
+                              streetAddressAdditional: ''
+                            mailingAddress:
+                              addressCity: Victoria
+                              addressCountry: CA
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V8W 1W4
+                              streetAddress: 1-940 Blanshard
+                              streetAddressAdditional: ''
+                            officer:
```

Comment:
Found an `email`  under some of the `officer` in the Postman collection. Not sure about it, but maybe you'd like to add them to here?
![image](https://github.com/user-attachments/assets/7389aeca-6aa2-4e83-9df9-8ea426b7437d)
![image](https://github.com/user-attachments/assets/64fc3d20-4964-4faa-8aba-bbc032e00a1b)